packages/apps/graph-client/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "next build",
+    "build": "rushx postinstall && next build",
```

Comment:
Do you know if this resolved the issue? 
The post-install script in this package does not depend on any other packages from the monorepo, so it should not fail due to missing builds. We should investigate why it failed during the post-install stage.

Reply:
It's not possible to run another `rush(x)` command during `rush build`. This will result in the error like "another rush command already running in this repository"